### ISSUE TITLE
lib/Specio/PartialDump.pm license

### ISSUE THREAD

**Issue by ppisar** [houseabsolute/Specio#17]
lib/Specio/PartialDump.pm file has two different license declarations:

~~~~
=head1 COPYRIGHT AND LICENSE

This software is copyright (c) 2008 by יובל קוג'מן (Yuval Kogman).

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.
~~~~

and 

~~~~
=head1 COPYRIGHT AND LICENSE

This software is Copyright (c) 2012 - 2019 by Dave Rolsky.

This is free software, licensed under:

  The Artistic License 2.0 (GPL Compatible)

The full text of the license can be found in the
F<LICENSE> file included with this distribution.
~~~~

Is that on purpose or is it a mistake?

**Comment by autarch** [comment]
That's definitely a mistake. The code in this module is basically copied from Yuval's `Devel::PartialDump`, so the first copyright is the right one. I'm not sure how to fix this - I may need to change how my dzil bundle works to accommodate this case.

**Comment by autarch** [comment]
Fixed in 0.46. Thanks!
